[2025-03-31 00:14:20,800: INFO: main: Welcome to our custom logg DataScience]
[2025-03-31 11:19:45,340: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-31 11:19:45,343: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-31 11:20:37,678: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-31 11:20:37,683: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-31 11:20:37,688: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-31 11:20:37,691: INFO: common: created directory at: artifacts]
[2025-03-31 11:20:37,693: INFO: common: created directory at: artifacts/data_ingestion]
[2025-03-31 11:20:38,635: INFO: 1445264634: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 23329
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "c69888a4ae59bc5a893392785a938ccd4937981c06ba8a9d6a21aa52b4ab5b6e"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 9032:2E622F:33E5BB:924298:67EA2D2E
Accept-Ranges: bytes
Date: Mon, 31 Mar 2025 05:50:38 GMT
Via: 1.1 varnish
X-Served-By: cache-hyd1100026-HYD
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1743400238.461091,VS0,VE294
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 1af513aef46366dd3edc036c3d51e1027f972c4d
Expires: Mon, 31 Mar 2025 05:55:38 GMT
Source-Age: 0

]
[2025-03-31 13:14:04,472: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-03-31 13:14:04,474: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-31 13:14:04,475: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-31 13:14:04,476: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-31 13:14:04,477: INFO: common: created directory at: artifacts]
[2025-03-31 13:14:04,477: INFO: common: created directory at: artifacts/data_ingestion]
[2025-03-31 13:14:05,255: INFO: data_ingestion: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 23329
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "c69888a4ae59bc5a893392785a938ccd4937981c06ba8a9d6a21aa52b4ab5b6e"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 9456:FA50A:28CDD2:7A70CC:67EA47C5
Accept-Ranges: bytes
Date: Mon, 31 Mar 2025 07:44:05 GMT
Via: 1.1 varnish
X-Served-By: cache-hyd1100025-HYD
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1743407046.529846,VS0,VE272
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 722df6a2453c129cea7bd951371d350632aa0a60
Expires: Mon, 31 Mar 2025 07:49:05 GMT
Source-Age: 0

]
[2025-03-31 13:14:05,266: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x==========x]
[2025-03-31 13:46:09,290: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-31 13:46:09,294: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-31 13:46:09,300: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-31 13:46:09,303: INFO: common: created directory at: artifacts]
[2025-03-31 13:46:09,304: INFO: common: created directory at: artifacts/data_validation]
[2025-03-31 14:04:05,777: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-03-31 14:04:05,779: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-31 14:04:05,780: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-31 14:04:05,782: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-31 14:04:05,783: INFO: common: created directory at: artifacts]
[2025-03-31 14:04:05,783: INFO: common: created directory at: artifacts/data_ingestion]
[2025-03-31 14:04:05,784: INFO: data_ingestion: File already exists]
[2025-03-31 14:04:05,789: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x==========x]
[2025-03-31 14:04:05,790: INFO: main: >>>>>> stage Data Validation Stage started <<<<<<]
[2025-03-31 14:04:05,791: ERROR: main: 'DataValidationTrainingPipeline' object has no attribute 'initiate_data_ingestion']
Traceback (most recent call last):
  File "C:\ML_OPS\DataScience_Project\Datascience_Project\main.py", line 25, in <module>
    obj.initiate_data_ingestion()
AttributeError: 'DataValidationTrainingPipeline' object has no attribute 'initiate_data_ingestion'
[2025-03-31 14:04:50,486: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-03-31 14:04:50,488: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-31 14:04:50,489: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-31 14:04:50,491: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-31 14:04:50,492: INFO: common: created directory at: artifacts]
[2025-03-31 14:04:50,492: INFO: common: created directory at: artifacts/data_ingestion]
[2025-03-31 14:04:50,493: INFO: data_ingestion: File already exists]
[2025-03-31 14:04:50,498: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x==========x]
[2025-03-31 14:04:50,499: INFO: main: >>>>>> stage Data Validation Stage started <<<<<<]
[2025-03-31 14:04:50,503: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-31 14:04:50,504: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-31 14:04:50,507: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-31 14:04:50,508: INFO: common: created directory at: artifacts]
[2025-03-31 14:04:50,509: INFO: common: created directory at: artifacts/data_validation]
[2025-03-31 14:04:50,529: INFO: main: >>>>>> stage Data Validation Stage completed <<<<<<

x==========x]
[2025-03-31 14:22:17,749: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-31 14:22:17,751: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-31 14:22:17,755: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-31 14:22:17,757: INFO: common: created directory at: artifacts]
[2025-03-31 14:22:17,759: INFO: common: created directory at: artifacts/data_transformation]
[2025-03-31 14:22:17,794: INFO: 3018882184: Splited data into training and test sets]
[2025-03-31 14:22:17,795: INFO: 3018882184: (1199, 12)]
[2025-03-31 14:22:17,800: INFO: 3018882184: (400, 12)]
[2025-03-31 14:42:12,497: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-03-31 14:42:12,501: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-31 14:42:12,503: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-31 14:42:12,506: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-31 14:42:12,507: INFO: common: created directory at: artifacts]
[2025-03-31 14:42:12,508: INFO: common: created directory at: artifacts/data_ingestion]
[2025-03-31 14:42:12,509: INFO: data_ingestion: File already exists]
[2025-03-31 14:42:12,519: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x==========x]
[2025-03-31 14:42:12,520: INFO: main: >>>>>> stage Data Validation Stage started <<<<<<]
[2025-03-31 14:42:12,523: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-31 14:42:12,525: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-31 14:42:12,533: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-31 14:42:12,534: INFO: common: created directory at: artifacts]
[2025-03-31 14:42:12,535: INFO: common: created directory at: artifacts/data_validation]
[2025-03-31 14:42:12,555: INFO: main: >>>>>> stage Data Validation Stage completed <<<<<<

x==========x]
[2025-03-31 14:42:12,556: INFO: main: >>>>>> stage Data Transformation Stage started <<<<<<]
[2025-03-31 14:42:12,560: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-31 14:42:12,564: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-31 14:42:12,568: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-31 14:42:12,569: INFO: common: created directory at: artifacts]
[2025-03-31 14:42:12,570: INFO: common: created directory at: artifacts/data_transformation]
[2025-03-31 14:42:12,603: INFO: data_transformation: Splited data into training and test sets]
[2025-03-31 14:42:12,603: INFO: data_transformation: (1199, 12)]
[2025-03-31 14:42:12,604: INFO: data_transformation: (400, 12)]
[2025-03-31 14:42:12,605: INFO: main: >>>>>> stage Data Transformation Stage completed <<<<<<

x==========x]
[2025-03-31 15:42:38,088: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-31 15:42:38,091: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-31 15:42:38,098: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-31 15:42:38,101: INFO: common: created directory at: artifacts]
[2025-03-31 15:42:38,103: INFO: common: created directory at: artifacts/model_trainer]
[2025-03-31 15:55:14,581: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-03-31 15:55:14,590: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-31 15:55:14,591: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-31 15:55:14,593: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-31 15:55:14,594: INFO: common: created directory at: artifacts]
[2025-03-31 15:55:14,594: INFO: common: created directory at: artifacts/data_ingestion]
[2025-03-31 15:55:14,595: INFO: data_ingestion: File already exists]
[2025-03-31 15:55:14,606: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x==========x]
[2025-03-31 15:55:14,607: INFO: main: >>>>>> stage Data Validation Stage started <<<<<<]
[2025-03-31 15:55:14,610: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-31 15:55:14,612: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-31 15:55:14,615: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-31 15:55:14,617: INFO: common: created directory at: artifacts]
[2025-03-31 15:55:14,618: INFO: common: created directory at: artifacts/data_validation]
[2025-03-31 15:55:14,637: INFO: main: >>>>>> stage Data Validation Stage completed <<<<<<

x==========x]
[2025-03-31 15:55:14,638: INFO: main: >>>>>> stage Data Transformation Stage started <<<<<<]
[2025-03-31 15:55:14,642: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-31 15:55:14,643: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-31 15:55:14,646: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-31 15:55:14,647: INFO: common: created directory at: artifacts]
[2025-03-31 15:55:14,650: INFO: common: created directory at: artifacts/data_transformation]
[2025-03-31 15:55:14,682: INFO: data_transformation: Splited data into training and test sets]
[2025-03-31 15:55:14,683: INFO: data_transformation: (1199, 12)]
[2025-03-31 15:55:14,684: INFO: data_transformation: (400, 12)]
[2025-03-31 15:55:14,686: INFO: main: >>>>>> stage Data Transformation Stage completed <<<<<<

x==========x]
[2025-03-31 15:55:14,686: INFO: main: >>>>>> stage Model Training Stage started <<<<<<]
[2025-03-31 15:55:14,690: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-31 15:55:14,691: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-31 15:55:14,694: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-31 15:55:14,695: INFO: common: created directory at: artifacts]
[2025-03-31 15:55:14,695: INFO: common: created directory at: artifacts/data_validation]
[2025-03-31 15:55:14,696: ERROR: main: 'ModelTrainer' object has no attribute 'validate_all_columns']
Traceback (most recent call last):
  File "C:\ML_OPS\DataScience_Project\Datascience_Project\main.py", line 49, in <module>
    obj.initiate_model_training()
  File "C:\ML_OPS\DataScience_Project\Datascience_Project\src\DataScience\pipeline\model_trainer_pipeline.py", line 17, in initiate_model_training
    data_validation.validate_all_columns()
AttributeError: 'ModelTrainer' object has no attribute 'validate_all_columns'
[2025-03-31 15:58:13,284: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-03-31 15:58:13,284: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-31 15:58:13,284: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-31 15:58:13,284: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-31 15:58:13,284: INFO: common: created directory at: artifacts]
[2025-03-31 15:58:13,284: INFO: common: created directory at: artifacts/data_ingestion]
[2025-03-31 15:58:13,284: INFO: data_ingestion: File already exists]
[2025-03-31 15:58:13,296: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x==========x]
[2025-03-31 15:58:13,297: INFO: main: >>>>>> stage Data Validation Stage started <<<<<<]
[2025-03-31 15:58:13,302: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-31 15:58:13,303: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-31 15:58:13,305: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-31 15:58:13,305: INFO: common: created directory at: artifacts]
[2025-03-31 15:58:13,305: INFO: common: created directory at: artifacts/data_validation]
[2025-03-31 15:58:13,324: INFO: main: >>>>>> stage Data Validation Stage completed <<<<<<

x==========x]
[2025-03-31 15:58:13,325: INFO: main: >>>>>> stage Data Transformation Stage started <<<<<<]
[2025-03-31 15:58:13,328: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-31 15:58:13,330: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-31 15:58:13,334: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-31 15:58:13,335: INFO: common: created directory at: artifacts]
[2025-03-31 15:58:13,335: INFO: common: created directory at: artifacts/data_transformation]
[2025-03-31 15:58:13,359: INFO: data_transformation: Splited data into training and test sets]
[2025-03-31 15:58:13,360: INFO: data_transformation: (1199, 12)]
[2025-03-31 15:58:13,360: INFO: data_transformation: (400, 12)]
[2025-03-31 15:58:13,361: INFO: main: >>>>>> stage Data Transformation Stage completed <<<<<<

x==========x]
[2025-03-31 15:58:13,362: INFO: main: >>>>>> stage Model Training Stage started <<<<<<]
[2025-03-31 15:58:13,366: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-31 15:58:13,367: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-31 15:58:13,369: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-31 15:58:13,370: INFO: common: created directory at: artifacts]
[2025-03-31 15:58:13,371: INFO: common: created directory at: artifacts/model_trainer]
[2025-03-31 15:58:13,373: ERROR: main: 'ModelTrainer' object has no attribute 'validate_all_columns']
Traceback (most recent call last):
  File "C:\ML_OPS\DataScience_Project\Datascience_Project\main.py", line 49, in <module>
    obj.initiate_model_training()
  File "C:\ML_OPS\DataScience_Project\Datascience_Project\src\DataScience\pipeline\model_trainer_pipeline.py", line 17, in initiate_model_training
    data_validation.validate_all_columns()
AttributeError: 'ModelTrainer' object has no attribute 'validate_all_columns'
[2025-03-31 16:02:40,059: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2025-03-31 16:02:40,062: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-31 16:02:40,064: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-31 16:02:40,066: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-31 16:02:40,066: INFO: common: created directory at: artifacts]
[2025-03-31 16:02:40,067: INFO: common: created directory at: artifacts/data_ingestion]
[2025-03-31 16:02:40,067: INFO: data_ingestion: File already exists]
[2025-03-31 16:02:40,074: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x==========x]
[2025-03-31 16:02:40,074: INFO: main: >>>>>> stage Data Validation Stage started <<<<<<]
[2025-03-31 16:02:40,077: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-31 16:02:40,078: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-31 16:02:40,082: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-31 16:02:40,082: INFO: common: created directory at: artifacts]
[2025-03-31 16:02:40,082: INFO: common: created directory at: artifacts/data_validation]
[2025-03-31 16:02:40,101: INFO: main: >>>>>> stage Data Validation Stage completed <<<<<<

x==========x]
[2025-03-31 16:02:40,103: INFO: main: >>>>>> stage Data Transformation Stage started <<<<<<]
[2025-03-31 16:02:40,111: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-31 16:02:40,112: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-31 16:02:40,114: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-31 16:02:40,115: INFO: common: created directory at: artifacts]
[2025-03-31 16:02:40,115: INFO: common: created directory at: artifacts/data_transformation]
[2025-03-31 16:02:40,152: INFO: data_transformation: Splited data into training and test sets]
[2025-03-31 16:02:40,152: INFO: data_transformation: (1199, 12)]
[2025-03-31 16:02:40,153: INFO: data_transformation: (400, 12)]
[2025-03-31 16:02:40,156: INFO: main: >>>>>> stage Data Transformation Stage completed <<<<<<

x==========x]
[2025-03-31 16:02:40,159: INFO: main: >>>>>> stage Model Training Stage started <<<<<<]
[2025-03-31 16:02:40,163: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-31 16:02:40,165: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-31 16:02:40,168: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-31 16:02:40,168: INFO: common: created directory at: artifacts]
[2025-03-31 16:02:40,170: INFO: common: created directory at: artifacts/model_trainer]
[2025-03-31 16:02:40,193: INFO: main: >>>>>> stage Model Training Stage completed <<<<<<

x==========x]
